LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 FRC Team 199: Deep Blue
```

Comment:
FRC Team 199 is not a legal entity so I don't think it can be the author. Either put your own name on it, or if you'd rather not for some reason, we can investigate alternatives. For example, you could send me an email saying you are giving me the copyright, so I can put my name on it.

Same would go for the other repos.

Reply:
Actually, you could also just use "CoolSpy3" as the name of the author.